integreat_compass/cms/views/interactions/vote_form_view.py

```diff
@@ -0,0 +1,104 @@
+from django.contrib import messages
+from django.db.models import Case, Count, When
+from django.shortcuts import redirect
+from django.utils.translation import gettext as _
+from django.views.decorators.http import require_POST
+from django.views.generic import TemplateView
+
+from ...constants import offer_version_states
+from ...decorators import permission_required
+from ...models import OfferVersion, Vote
+
+
```

Comment:
```suggestion

@method_decorator(permission_required("cms.add_vote"), name="dispatch")
```
As mentioned in #56, I think it makes sense to use the default Django permissions `add_*` etc instead of custom `vote_on_offer` if they essentially mean the same.
Also, we probably can enforce these permissions for the get-view as well, since it does not make sense to render the voting view if the user does not have the permission to cast a vote?